CommonClient.py

```diff
@@ -1095,7 +1095,7 @@ async def server_auth(self, password_requested: bool = False):
             if password_requested and not self.password:
                 await super(TextContext, self).server_auth(password_requested)
             await self.get_username()
-            await self.send_connect()
+            await self.send_connect(game="")
 
         def on_package(self, cmd: str, args: dict):
```

Comment:
i think the proper fix is to just remove this method since that's all it does

Reply:
can't close my comment but i completely missed that common client needs this attribute set to function in any proper respect